cmd/rooms_api/wire_gen.go

```diff
@@ -1,7 +1,8 @@
 // Code generated by Wire. DO NOT EDIT.
 
 //go:generate go run github.com/google/wire/cmd/wire
-//+build !wireinject
+//go:build !wireinject
```

Comment:
Every time we run "make generate" this is created, but do we actually want to merge this? What does it mean?